tests/conftest.py

```diff
@@ -25,6 +25,17 @@
 ## PyTest Hooks
 ##########################################################################
 
+def docline(obj):
+    """
+    Returns the first line of the object's docstring or None if
+    there is no __doc__ on the object.
+    """
+    if not obj.__doc__:
+        return None
+    lines = list(filter(None, obj.__doc__.split("\n")))
+    return lines[0].strip()
```

Comment:
Something we've needed for a while, takes only the first line of the docstring which converts our test output as follows:

```
Test the dataset loading functions (tests/test_datasets/test_loaders.py)
    Broadly: test each of the dataset loaders to ensure that they are valid
    for their particular type of dataset and that they return X and y by
    default. Then test their shape to ensure that the dataset hasn't changed
    unexpectedly between versions. See ``test_load_concrete`` for a sketch.
    Final tests with parametrize test all loaders against Base classes.
    Make sure you scroll to the bottom and implement your loader in the
    correct test batch! (tests/test_datasets/test_loaders.py)
    [PASS]  Test loading the concrete regression dataset
    [PASS]  Test loading the energy multi regression dataset
    [PASS]  Test loading the credit binary classification dataset
    [PASS]  Test loading the occupancy binary classification dataset
```

To:

```
```
Test the dataset loading functions (tests/test_datasets/test_loaders.py)
    [PASS]  Test loading the concrete regression dataset
    [PASS]  Test loading the energy multi regression dataset
    [PASS]  Test loading the credit binary classification dataset
    [PASS]  Test loading the occupancy binary classification dataset
```

Reply:
Much nicer!